x-pack/test/reporting_api_integration/config.js

```diff
@@ -6,34 +6,35 @@
 
 import { esTestConfig, kbnTestConfig, kibanaServerTestUser } from '@kbn/test';
 import { format as formatUrl } from 'url';
-import { ReportingAPIProvider } from '../services';
+import { ReportingAPIProvider } from './services';
 
 export default async function({ readConfigFile }) {
-  const apiConfig = await readConfigFile(require.resolve('../../api_integration/config.js'));
+  const apiConfig = await readConfigFile(require.resolve('../api_integration/config'));
+  const functionalConfig = await readConfigFile(require.resolve('../functional/config')); // Reporting API tests need a fully working UI
 
   return {
     servers: apiConfig.get('servers'),
-    junit: { reportName: 'X-Pack Reporting Generate API Integration Tests' },
-    testFiles: [require.resolve('../api/generate')],
+    junit: { reportName: 'X-Pack Reporting API Integration Tests' },
+    testFiles: [require.resolve('./reporting')],
     services: {
       ...apiConfig.get('services'),
       reportingAPI: ReportingAPIProvider,
     },
     kbnTestServer: {
       ...apiConfig.get('kbnTestServer'),
       serverArgs: [
-        '--logging.events.log',
-        '["info","warning","error","fatal","optimize","reporting"]',
```

Comment:
This logging config param was removed, because it isn't the correct way to see debug logs for just Reporting. This is the case since the Reporting config migration.